### ISSUE TITLE
Document how to use Django with ASGI

### ISSUE THREAD

**Issue by owais** [open-telemetry/opentelemetry-python-contrib#1051]
ASGI support for Django was added in the PR #391 but we never documented the usage. 

Users may have to install `opentelemetry-instrumentation-django[asgi]` in order to take advantage of asgi support. This should be documented at https://opentelemetry-python-contrib.readthedocs.io/en/latest/instrumentation/django/django.html

**Comment by rahuldimri** [comment]
@owais can I work on this ? Please assign this to me.

**Comment by rahuldimri** [comment]
Thanks for assigning this to me. I am new to OpenSource community. If you can help me initially that would be great.
Just wanted to know do you have any template for updating the document ?.

**Comment by lzchen** [comment]
You can make your doc addition to the [docstrings](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/instrumentation/opentelemetry-instrumentation-django/src/opentelemetry/instrumentation/django/__init__.py#L16) of the django instrumentation. Your docs will be reflected in readthedocs if you do this. Follow the structuring and wording of the other types of configuration, I would name the header something like `Asgi Configuration` or something.

**Comment by Abhishek-569** [comment]
I want to work on this issue. Is there any example or use cases  on what exactly ASGI support for Django help the users with ?

**Comment by srikanthccv** [comment]
Your question is unclear; how are "use cases" relevant to this issue?

**Comment by Abhishek-569** [comment]
Sorry if my question was not clear before. I am asking like this below `for example` on this [page ](https://opentelemetry-python-contrib.readthedocs.io/en/latest/instrumentation/django/django.html)  for Django instrumentation we have to give one example or use case for the proper documentation of using  Django with asgi. Please tell me if something is wrong with my proposal as this is my first issue on this project.
![image](https://user-images.githubusercontent.com/64806938/233091527-e9b3857b-30e2-4b93-9c74-dca46162a806.png)

**Comment by srikanthccv** [comment]
`EXCLUDE_URLS` has examples since it's a configuration option. This issue is about documenting that Django instrumentation supports ASGI.

**Comment by Abhishek-569** [comment]
Okay. we should add this issue in this [page](https://opentelemetry-python-contrib.readthedocs.io/en/latest/instrumentation/django/django.html), right?

**Comment by srikanthccv** [comment]
Yes, you need to update the docstrings, but this issue is already assigned. @rahuldimri please unassign yourself if you are not working on this.

**Comment by Abhishek-569** [comment]
Okay can tell me where I can learn more on` How to use Django with asgi `helps in open-telemetry?

**Comment by baravkareknath** [comment]
Hi Abhishek-569 can you assign this documentation work to me if you are not working on the same.
